jvm-agent/src/main/org/jetbrains/kotlinx/lincheck/transformation/TransformationUtils.kt

```diff
@@ -651,6 +651,24 @@ internal fun isJavaUtilArraysClass(className: String): Boolean =
 internal fun isJavaLangAccessClass(className: String): Boolean =
     className == "jdk.internal.access.JavaLangAccess"
 
+/**
+ * Checks if the given class name belongs to the IntelliJ runtime debugger agent package.
+ */
+internal fun isIntellijRuntimeDebuggerAgentClass(className: String) =
+    className.startsWith("com.intellij.rt.debugger.agent")
+
+/**
+ * Checks if the given class name belongs to the IntelliJ runtime coverage agent package.
+ */
+internal fun isIntellijRuntimeCoverageAgentClass(className: String) =
+    className.startsWith("com.intellij.rt.coverage")
+
+/**
+ * Checks if the given class name belongs to the IntelliJ runtime agents.
+ */
+internal fun isIntellijRuntimeAgentClass(className: String) =
```

Comment:
Can you please search for these package names in entering codebase,I think this is also user in some code related to snapshot. That place should be rewritten using this function.

Apart from that lgtm

Reply:
for snapshot it was removed as it seems, in one of the old PRs